python/desc/monitor/dbConnection.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+from lsst.sims.catalogs.generation.db import DBObject
+
+class dbInterface(object):
+
+    def __init__(self):
```

Comment:
@jbkalmbach Regarding giving users the ability to select the database, host, and port, I guess the minimal thing to do would be to make these arguments to the `dbInterface` constructor here and also in the `LightCurve.build_lightcurve_from_db` method below.  To be explicit, the code here would be
```
    def __init__(self, database, host, port, driver):
        self._dbo = DBObject(database=database, host=host, port=port, driver=driver)
```